serverless/configuring-knative-serving-autoscaling.adoc

```diff
@@ -0,0 +1,14 @@
+[id="configuring-knative-serving-autoscaling"]
+= Configuring Knative Serving autoscaling
+include::modules/common-attributes.adoc[]
+:context: configuring-knative-serving-autoscaling
+
+Red Hat OpenShift Serverless provides capabilities for automatic pod scaling, including scaling inactive pods to zero, by enabling the Knative Serving autoscaling system in an {product-title} cluster.
```

Comment:
Not critical, but I'd recommend to create an attribute for the product name.

Reply:
Are there any docs about best practices for this?
I've used `:FeatureName: Red Hat OpenShift Serverless` in one of the assemblies as it's required for the tech preview module, but idk if we add the same in each module and use {FeatureName}, or is there a way we add this to the centralised attributes for OpenShift so we can use it across all files, like {product-title}?

Reply:
I don't know if we have it documented anywhere, but I don't see why you couldn't just put it in the common-attributes.adoc file with the other OpenShift attributes.

I think CNV might have their own attributes file too, so you could probably do whatever works best for you guys!